backend/src/modules/search/services/search.service.ts

```
@@ -118,20 +90,25 @@ export class SearchService {
-  static async searchContent(searchText: string) {
-    try {
-      if (!searchText) {
-        return { documents: [], nextStartingPoint: null };
-      }
+  static async searchContents(searchText: string) {
+    if (!searchText) {
+      return { documents: [], nextStartingPoint: null };
+    }
+
+    const client = SearchService.getAlgoliaClient();
+    const index = client.initIndex(this.ALGOLIA_INDEX_NAME);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)